### ISSUE TITLE
ipmctl help says its program name is ipmctl.efi in linux

### ISSUE THREAD

**Issue by RobertCElliott** [intel/ipmctl#9]
$ sudo ipmctl help

Intel(R) Optane(TM) DC Persistent Memory Command Line Interface

    Usage: ipmctl.efi <verb>[<options>][<targets>][<properties>]

due to:
```
DcpmemPkg/cli/NvmDimmCli.h:#define EXE_NAME               L"ipmctl.efi"
DcpmemPkg/cli/NvmDimmCli.c:    Print(FORMAT_STR_SPACE FORMAT_STR_NL_NL L"    Usage: " FORMAT_STR L" <verb>[<options>][<targets>][<properties>]\n\nCommands:\n", PRODUCT_NAME, APP_DESCRIPTION, EXE_NAME);

```

**Comment by juston-li** [comment]
Fixed in 7e58254d5e442143a024e5047f7791ddc4b03f04

Thanks!

**Comment by RobertCElliott** [comment]
In Windows you don't need to type .exe in program names, so I recommend using "ipmctl" in that case.

I think the same is true for UEFI.  It's definitely not needed if the command is built-in.  If so, all three environments should just use "ipmctl".
